examples/swarmion-full-stack/frontend/app/package.json

```diff
@@ -27,12 +27,12 @@
     "@mui/lab": "5.0.0-alpha.114",
```

Comment:
Could you rename the commit to `examples(full-stack): ...` so that it appears the right way in the release motes?